contents/javascript/Javascript-framework-VueJs.md

```diff
@@ -151,18 +153,147 @@ VueJs is a relatively new JavaScript framework as compared to Angular and React.
 2. **Language barriers:**<br/>
 A majority of users of VueJs are the Chinese as VueJs is developed by a Chinese American. He is supportive of the Chinese community and hence a lot of the existing plugins are written in Chinese. There might be some language barriers for an English speaking developer seeking for VueJs resources.
 
-#### Detailed Comparison of VueJs with other JavaScript frameworks can be found from:
+-----
+### Main concepts needed in VueJs
```

Comment:
`Vue Features`?

Also, move this above the `Advantages and Disadvantages of VueJs` section.